.goreleaser.yml

```diff
@@ -33,9 +33,16 @@ archives:
 checksum:
   name_template: "doctl-{{ .Version }}-checksums.sha256"
 
+dockers:
+- dockerfile: Dockerfile.goreleaser
+  image_templates:
+  - "hilz/doctl:latest"
```

Comment:
Should this be DO's account instead of yours?

Reply:
ayup, fixing :)